Error: Cannot find module 'fs-extra'

I install via npm and get the error above, if i go and install fs-extra i get the following

```
Error: Cannot find module '../package.json'
Require stack:
- /usr/lib/node_modules/wc3maptranslator/dist/src/cli.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1421:15)
    at defaultResolveImpl (node:internal/modules/cjs/loader:1059:19)
    at resolveForCJSWithHooks (node:internal/modules/cjs/loader:1064:22)
    at Module._load (node:internal/modules/cjs/loader:1227:37)
    at TracingChannel.traceSync (node:diagnostics_channel:328:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:245:24)
    at Module.require (node:internal/modules/cjs/loader:1504:12)
    at require (node:internal/modules/helpers:152:16)
    at Object.<anonymous> (/usr/lib/node_modules/wc3maptranslator/dist/src/cli.js:34:24)
    at Module._compile (node:internal/modules/cjs/loader:1761:14) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '/usr/lib/node_modules/wc3maptranslator/dist/src/cli.js' ]
}
```

--- a/package.json
+++ b/package.json
@@ -54,6 +54,7 @@
     "dependencies": {
         "chalk": "^5.6.2",
         "commander": "^14.0.2",
+        "fs-extra": "^11.3.5",
         "ieee754": "^1.2.1",
         "intn": "^1.0.0",
         "round-to": "^7.0.0"
@@ -66,7 +67,6 @@
         "coveralls": "^3.1.0",
         "eslint": "^9.39.2",
         "eslint-plugin-github": "^6.0.0",
-        "fs-extra": "^11.3.3",
         "source-map-support": "^0.5.19",
         "ts-node": "^9.1.1",
         "typescript": "^4.2.2",

--- a/src/cli.ts
+++ b/src/cli.ts
@@ -5,11 +5,14 @@
 import chalk from 'chalk';
 import { program } from 'commander';
 import * as fs from 'fs-extra';
+import { createRequire } from 'node:module';
 import path from 'node:path';
-import { version } from '../package.json';
 import { JsonResult, WarResult, ITranslator, VersionError } from './CommonInterfaces';
 import { ObjectsTranslator } from './index';
 import translatorMappings from './TranslatorMappings';
+
+const requirePackageJson = createRequire(__filename);
+const { version } = requirePackageJson('../../package.json') as { version: string };
 
 const knownWarFiles = translatorMappings.map((mapping) => mapping.warFile);
 const knownJsonFiles = translatorMappings.map((mapping) => mapping.jsonFile);

